compiler/haskell-ide-core/BUILD.bazel

```
@@ -64,7 +65,7 @@ da_haskell_library(
 
 da_haskell_binary(
     name = "haskell-ide-core-demo",
-    srcs = glob(["test/**/*.hs"]),
+    srcs = glob(["test/Demo.hs"]),
```

```suggestion
    srcs = ["test/Demo.hs"],
```
If you're listing sources explicitly, you don't really need to glob anymore. Also if the source file is renamed, then `glob` would return an empty list, which would produce a more confusing error message than listing the file explicitly.